docs/source/apis/api_leveling.md

```diff
@@ -0,0 +1,51 @@
+# Llama Stack API Stability Leveling
+
+In order to provide a stable experience in Llama Stack, the various APIs need different stability levels indicating the level of support, backwards compatability, and overall production readiness.
+
+## Different Levels
+
+### v1alpha1
```

Comment:
`/alpha/v1` or `/beta/v1/` feel slightly more readable?

Reply:
I don't think `v1alpha2` would exist. If it helps clear it up I can switch both to `v1alpha` and `v1beta` without the trailing 1. I agree, It could be nice to extend this in the case of a `v2` api.